src/euroeval/prompt_templates/sentiment_classification.py

```diff
@@ -78,15 +78,19 @@
         "meelestatuse järgi. Võimalikud vastused: {labels_str}. Muud vastused "
         "ei ole lubatud.",
     ),
-    PT: PromptConfig(
+    PL: PromptConfig(
```

Comment:
why PT is removed?

Reply:
Cool call, must have slipped. Has been added back on the main branch now, thanks!